app/services/paypal_service/api/payments.rb

```diff
@@ -266,6 +266,14 @@ def retry_and_clean_tokens(clean_time_limit)
     # Reusable bits of the API operations
     #
 
+    def stop_retrying_token?(response, token_created_at, clean_time_limit)
+      if (token_created_at < clean_time_limit && (response[:data] == nil || response[:data][:error_code] != :"payment-review"))
```

Comment:
Would it be better to return just the result of the comparison? (or use variable for the sake of readability and return that)